### DISCUSSION TITLE
Users can't seem to download recipes

### DISCUSSION THREAD

**Discussion by CiaranGruber** [nextcloud/cookbook#2440] [FAQ and user questions]
I've created a Recipes folder which I've then shared with all the users on my Nextcloud instance (I'm the admin). However, I've found that only I can seem to download recipes and none of my users can download recipes and must enter them manually.

Is there a setting I could change to allow users to download recipes, potentially with a list of approved websites if security is a concern

**Comment by christianlupus** [comment]
Can the other users access the same recipes? Can you check with a recipe you have successfully imported?

There is no blacklist or such implemented. If it is not working, one of the following things might be the cause of it:

1. The URL is invalid or the server is not able to fetch/download the actual URL (seems trivial but has happened)
2. The downloaded file is not parsable. The provider is not able or willing to share schema.org meta data.
3. The file cannot be saved on the hard drive. This is more of a NC core issue
4. The DB get corrupted for some reason. This should be fixed (from the perspective of Cookbook) by a rescan of the recipes folder.

If the users are generally able to create new recipes, this rules out 3 and 4. If you managed to import it, 1 and 2 seems to work as well. So, I am a bit startled that this would cause trouble.

The only thing, I might see here: How did you share the folder? Was this a personal share or a group folder or something different?

Could one of your users test the following:
1. Switch the recipe folder to something completely locally. Nothing shared or such, just a bare folder in the user's files.
2. Try to import the offending recipe.
3. Switch back to the shared folder (to restore the state)
